graspologic/embed/lse.py

```diff
@@ -167,12 +169,29 @@ def fit(self, graph: Union[np.ndarray, networkx.Graph], y=None):
         self._reduce_dim(L_norm)
 
         # for out-of-sample
-        inv_eigs = np.diag(1 / self.singular_values_) / (
-            self.n_features_in_ / self.n_components_
-        )
+        inv_eigs = np.diag(1 / self.singular_values_)
+
         self._pinv_left = self.latent_left_ @ inv_eigs
         if self.latent_right_ is not None:
             self._pinv_right = self.latent_right_ @ inv_eigs
 
         self.is_fitted_ = True
         return self
+
+    def compute_oos_prediction(self, X, directed):
+        # workhorse code
+        if not directed:
+            if X.ndim == 1:
+                return X @ self._pinv_left / np.sum(X)
+            else:
+                return np.transpose(np.transpose(X @ self._pinv_left) /
+                                    np.sum(X, axis=1))
```

Comment:
can do `X.T` instead of `np.transpose(X)`, probably cleaner

Reply:
for sure